src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/card/DefaultStyles.kt

```diff
@@ -86,6 +95,13 @@ class DefaultStyles : Stylesheet() {
             effect = DropShadow(2.0, 4.0, 6.0, AppTheme.colors.lightBackground)
         }
 
+        val defaultCardButtonHover = mixin {
+            textFill = AppTheme.colors.white
+            defaultCardButtonIcon {
```

Comment:
I think this `defaultButtonIcon` is misleading because there isn't actually any icon associated with this. Also these DefaultStyles are just meant to be for the items in the widgets directory. I'd messed it up in previous work and have been meaning to refactor the places that it got imported inside the app directory (my bad). Let's not compound my mistake and move styles for things that don't live inside the widgets directory out of here into a more appropriate location.